src/FuseGPUThreadLoops.cpp

```diff
@@ -437,6 +437,8 @@ class ExtractSharedAndHeapAllocations : public IRMutator {
             }
         }
 
+        debug(1) << "== Changing allocation " << op->name << " to " << alloc.name <<"\n";
```

Comment:
There are some debug(1)s that should be removed (or demoted to debug(3)) before we merge this.